assets/stylesheet/styles.css

```diff
@@ -0,0 +1,824 @@
+
+#figure2 {
+  width: 100%;
+  margin: 0 auto;
+}
+
+#figure2 img {
+  width: 102%;
+  height: auto;
+}
+
+#author {
+  width: 46.875vw;
+  height: auto;
+}
+
+/* ======= ===========   ================ */
+
+/* <aside> */
+
+/* =====   ==========     ================ */
+
+/* MAIN aside */
+
+.main-aside {
+  width: 13.125rem;
+  height: 59.23rem;
+  font-family: 'NYTImperial', sans-serif;
+  font-size: 0.85rem;
+  position: absolute;
+  top: 17%;
+  left: 82%;
+}
+
+.aside-container {
+  display: grid;
+  grid-template-columns: 40% 60%;
+  margin-top: 1.25rem;
+}
+
+.aside-img {
+  width: 80px;
+  height: 77px;
+  border: 1px solid black;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  vertical-align: middle;
+}
+
+.aside-container .aside-img {
+  cursor: pointer;
+}
+
+.aside-container .aside-text {
+  cursor: pointer;
+}
+
+#editor {
+  border-bottom: 1px solid rgb(222, 222, 222);
+  font-family: 'NYTFranklin', sans-serif;
+  font-size: 1.094vw;
+  line-height: 1.094vw;
+  letter-spacing: 0.022vw;
+  padding: 0 0 0.938vw;
+  margin: 0 0 1.172vw;
+}
+
+.img-1 {
+  background-image: url(../26mag-Relampago-Web-Image-square640.jpg);
+}
+
+.img-2 {
+  background-image: url(../26mag-Migratiion-Images-square640.jpg);
+}
+
+.img-3 {
+  background-image: url(../22fakedeath-square640-v3.jpg);
+}
+
+.img-4 {
+  background-image: url(../sc4.png);
+}
+
+/* ======= ===========   ================ */
+
+/* ======= ===========   ================ */
+
+/* footer */
+
+/* =====   ==========     ================ */
+
+#footer-main {
+  width: 95%;
+  box-sizing: border-box;
+  margin: 0 auto;
+  margin-top: 2.563vw;
+  color: black;
+}
+
+#footer-header {
+  margin: 0 0.5vw;
+  border-top: 4px double rgb(226, 226, 226);
+  padding-top: 0.781vw;
+}
+
+#footer-header img {
+  height: 2.344vw;
+  position: absolute;
+  left: 2.7%;
+  cursor: pointer;
+}
+
+#footer-main-sect {
+  width: 100%;
+  display: grid;
+  grid-template-columns: 16.66% 16.66% 16.66% 16.66% 16.66% 16.66%;
+}
+
+#more-section {
+  border-right: 0.5px solid rgb(222, 222, 222);
+}
+
+.subs {
+  width: 15.484vw;
+  position: absolute;
+  left: 82.6%;
+}
+
+.section {
+  font-family: 'NYTFranklin', sans-serif;
+  font-size: 1.094vw;
+  line-height: 1.25vw;
+  padding: 0.781vw 0;
+}
+
+.section ul {
+  list-style: none;
+  padding: 0;
+}
+
+.img-icon {
+  max-width: 16px;
+  max-height: 16px;
+}
+
+#top-small {
+  padding-top: 0.625rem;
+}
+
+#last-small {
+  border-bottom: 0.5px solid rgb(222, 222, 222);
+  padding-bottom: 0.234vw;
+}
+
+#footer-b {
+  width: 90vw;
+  height: 5.156vw;
+  margin-top: -0.391vw;
+  border-top: 0.094rem solid rgb(218, 215, 215);
+  font-family: 'NYTFranklin', sans-serif;
+  font-size: 0.859vw;
+  line-height: 1.563vw;
+  padding: 0.781vw 0;
+  text-align: center;
+}
+
+/* YT figure */
+
+#YT-video {
+  margin-block-start: 0;
+  margin-block-end: 0;
+  margin-inline-start: 0;
+  margin-inline-end: 0;
+}
```

Comment:
There are a few CSS errors at this point kindly fix them.
![warnings css](https://user-images.githubusercontent.com/32361963/88902034-4d0feb80-d25a-11ea-9a8e-e4ffc9257876.jpg)